### ISSUE TITLE
Message Link detected as wrong link even if it correct

### ISSUE THREAD

**Issue by imagigabacon** [discohook/discohook#29]
when i gonna edit the message, it said the message link is wrong. When i tried replacing the same message link it still said wrong.
![image](https://github.com/user-attachments/assets/bd7db8aa-95e0-4029-85dd-f47fd7fafde8)
![image](https://github.com/user-attachments/assets/75aa308a-8cad-4c5f-92fc-fc59b097330b)

**Comment by imagigabacon** [comment]
More: i clicked on the webhook and clicked "Set Message Link" it still said wrong

**Comment by shayypy** [comment]
Although discohook re-interprets this error to make it easier to understand, the underlying cause is always an error from Discord with code 10008 (unknown message), and you can see this by clicking the red octagon on the left. Discord returns this for requests to the `/webhooks/:id/:token/messages/:message_id` route when the message represented by `message_id` was not sent by the webhook in the channel that was searched. If a thread ID is specified, the thread is the channel that is searched, otherwise it's the webhook's own channel ID. This is why Discohook prompts you to set a thread ID if necessary.

TL;DR this is not a Discohook bug and you should double check that the webhook and message match
